src/llfs/packed_array.hpp

```diff
@@ -116,6 +123,22 @@ struct PackedArray {
       out << "} PackedArray;" << std::endl;
     };
   }
+
+  void initialize_size_in_bytes(usize size_in_bytes)
+  {
+    this->size_in_bytes = size_in_bytes;
+    this->flags = static_cast<Flags>(this->flags | kSizeInBytesSet);
+  }
+
+  bool has_size_in_bytes() const
+  {
+    return (this->flags & kSizeInBytesSet);
+  }
+
+  Optional<usize> get_size_in_bytes() const
+  {
+    return this->size_in_bytes;
```

Comment:
This should return `None` if `this->has_size_in_bytes()` is `false`.

Reply:
That will be better. I have the flag checked in TurtleDB level. But putting it here is better. Will change it ...

Reply:
Thanks!